python/tests/test-1-vector.py

```diff
@@ -321,6 +322,7 @@ def test_199(ceed_resource):
     ceed = libceed.Ceed(ceed_resource)
 
     vec = ceed.Vector(12)
+    vec.set_value(0.0)
     with vec.array(4, 3) as x:
```

Comment:
Should we allow write/invalid access for Python users (wrapping `GetArrayWrite`) so initialization is not always required (like using `numpy.empy`)?

Reply:
I'm not sure. I definitely do not want to do it for the Rust interface. Python is more 'wild west' so I suppose I could go either way.

Reply:
Yeah, I think we should provide write access to Python, since we "accidentally" allowed it before (`GetArray` with uninitialized data was okay), and indeed, that is what these tests were doing.  I don't think we should force Python users to pay an unnecessary initialization cost when other standard Python computing libraries (like `numpy`) allow use of uninitialized data.  Of course, I don't know how often Python users will be manipulating `CeedVector`s directly, outside of other libCEED components, but I'd still vote for adding it to the Python API.

Reply:
I added `vector.array_write()`